test/e2e/storage/external/external.go

```diff
@@ -302,6 +309,34 @@ func (d *driverDefinition) GetDynamicProvisionStorageClass(e2econfig *testsuites
 	return testsuites.GetStorageClass(sc.Provisioner, sc.Parameters, sc.VolumeBindingMode, f.Namespace.Name, "e2e-sc")
 }
 
+func (d *driverDefinition) GetTimeouts() *framework.TimeoutContext {
+	timeouts := framework.NewTimeoutContextWithDefaults()
+	if d.Timeouts != nil {
+		c := make(map[string]time.Duration)
+		for k, v := range d.Timeouts {
+			duration, err := time.ParseDuration(v)
+			if err != nil {
+				// We can't use ExpectNoError() because his method can be called out of an It(),
+				// so we simply log the error and return the default timeouts.
+				klog.Errorf("Could not parse duration: %v", err)
```

Comment:
Can you also specify the key here for debugging: "Could not parse duration %v for key %s"